keyboard.lua

```diff
@@ -10,24 +10,53 @@ keyboard.key_map = {
 }
 
 function keyboard.move(time, obj)
-    -- Move object via wasd with force
-    local movementForce = 300
+    -- Standard amount of force to base movement off of
+    local movementForce = 100
+
+    -- Variables that allows the function to know which way the spaceship
+    -- is pointing and to move it in that direction.
+    local topx, topy = objects.square.body:getWorldPoint(0, -25)
+    local ratiox = (topx - obj.body:getX()) * 0.04
+    local ratioy = (topy - obj.body:getY()) * 0.04
+
+    -- Fly spaceship forwards and backwards
     if love.keyboard.isDown('w') then
-        obj.body:applyForce(0, -movementForce)
+        obj.body:applyForce(ratiox * movementForce, ratioy * movementForce)
     end
     if love.keyboard.isDown('s') then
+        obj.body:applyForce(ratiox * movementForce * -0.25, ratioy * movementForce * -0.25)
+    end
+
+    -- Steer spaceship left and right
+    local angVel = player.body:getAngularVelocity()
+    local maxAngVel = 2
+    if angVel > -maxAngVel and angVel < maxAngVel then
+        if love.keyboard.isDown('a') then
```

Comment:
Consider using [`love.keyboard.isScancodeDown()`](https://love2d.org/wiki/love.keyboard.isScancodeDown) instead. I don't have a qwerty keyboard so these bindings don't make sense trying to play on my keyboard but the scancode reference the physical switch locations regardless of what letter is mapped to that key in the OS.